### PR TITLE
Add 2020Q1 report for RISC-V

### PR DESCRIPTION
**Pull request by mhorne**
I've asked other RISC-V committers to have a look at the PR. Please hold off on merging until they have had the chance.

### CODE REVIEW

File: 2020q1/riscv.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Contact: John Baldwin <jhb@FreeBSD.org>  
+Contact: Kristof Provost <kp@FreeBSD.org>  
+Contact: Philip Paeps <philip@FreeBSD.org>  
+
+It has been a year since the RISC-V project's last status report. In that time,
+the RISC-V port has benefited from increased attention, and received
+improvements of all kinds.
+
+The RISC-V project has brought in two new src committers. We'd like to welcome
+James Clarke (jrtc27@), who is a member of CheriBSD, and Nick O'Brian (nick@) of
```
Comment (jrtc27) [inline 2020q1/riscv.md:16]: Now:
```suggestion
Jessica Clarke (jrtc27@), who is a member of CheriBSD, and Nick O'Brian (nick@) of
```

**Comment by jrtc27** [comment]
Is it worth mentioning that the DARPA SSITH program is funding a bunch of the CheriBSD and thus FreeBSD work, and mentioning the GFE hardware (an SoC with one of Rocket, BOOM, Flute or Toooba (a fork of MIT's RiscyOO) at its core) that forms part of the program?

File: 2020q1/riscv.md (hunk 1 above)
Comment (mhorne) [reply 2020q1/riscv.md]: Thank you, updated.

**Comment by mhorne** [comment]
> Is it worth mentioning that the DARPA SSITH program is funding a bunch of the CheriBSD and thus FreeBSD work, and mentioning the GFE hardware (an SoC with one of Rocket, BOOM, Flute or Toooba (a fork of MIT's RiscyOO) at its core) that forms part of the program?

Do you mean to say that we don't know which of those cores forms the basis of their hardware? Or that they have implementations for each?

I have listed DARPA as a sponsor, but we can expand that to a separate paragraph if you think it's worthwhile. I'm not attached to any of those programs so I would not be the best to describe them.

**Comment by jrtc27** [comment]
All of them are supported, with the following matrix:

Language \ Pipeline | In-order | Out-of-order
--- | --- | ---
**Bluespec** | Flute | Toooba (RiscyOO)
**Chisel** | Rocket | BOOM

**Comment by jrtc27** [comment]
@rwatson may have opinions on this.

File: 2020q1/riscv.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Contact: John Baldwin <jhb@FreeBSD.org>  
+Contact: Kristof Provost <kp@FreeBSD.org>  
+Contact: Philip Paeps <philip@FreeBSD.org>  
+
+It has been a year since the RISC-V project's last status report. In that time,
+the RISC-V port has benefited from increased attention, and received
+improvements of all kinds.
+
+The RISC-V project has brought in two new src committers. We'd like to welcome
+Jessica Clarke (jrtc27@), who is a member of CheriBSD, and Nick O'Brian (nick@)
```
Comment (nickisobrien) [inline 2020q1/riscv.md:16]: O'Brien

**Comment by kprovost** [comment]
LGTM

**Comment by debdrup** [comment]
I'll happily wait up until next the end of the day of the 7th of April, to give everyone who's going to review it a chance, since I'm fairly sure I only remember a subset of the people involved with RISC-V :)

**Comment by debdrup** [comment]
So, is this lovely report ready to be merged? :)

**Comment by mhorne** [comment]
> So, is this lovely report ready to be merged? :)

I'm happy with it. @jrtc27 if you'd like to add a section mentioning the DARPA SSITH program, please feel free.

**Comment by debdrup** [comment]
Merging after OK from @mhorne on IRC. Thanks so much to everyone :)

**Pull request merged**